client/a8c-for-agencies/sections/purchases/licenses/license-preview/index.tsx

```diff
@@ -104,11 +104,13 @@ export default function LicensePreview( {
 	const licenseState = getLicenseState( attachedAt, revokedAt );
 	const domain = siteUrl && ! isPressableLicense ? getUrlParts( siteUrl ).hostname || siteUrl : '';
 
+	const isClientSite = isAutomatedReferralsEnabled && referral;
```

Comment:
Maybe using isClientLicense is more appropriate here than using a isClientSite if this is a license link? wdyt?

Reply:
Thanks for the review, @jkguidaven. Makese total sense! I have made the changes accordingly: https://github.com/Automattic/wp-calypso/pull/94814/commits/5bd035523ba0ebf0d21623e943f23a2961c8bc1e